### PR TITLE
C

### PR DESCRIPTION
**Pull request by frite**
This PR did the following:
- Added support for parsing/calculating complexity for C Language. 
- Minor updates to README.md
- Minor updates to Makefile to ensure that git submodules are pulled.

@long2ice can you take a look and let me know? 

Regards

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,8 @@ deps: clean
 	poetry install
 
 style: deps
+	git submodule init
```
Comment (long2ice) [inline Makefile:11]: This should add under `deps`

**Review by long2ice — APPROVED** [review]
Looks all good! Just need little change.

File: README.md [hunk 2]
```diff
@@ -12,20 +12,22 @@ Current supported languages:
 - [Go](https://github.com/tree-sitter/tree-sitter-go)
 - [Python](https://github.com/tree-sitter/tree-sitter-javascript)
 - [JavaScript](https://github.com/tree-sitter/tree-sitter-python)
```
Comment (long2ice) [inline README.md:14]: Could you please fix this? `Python` and `JavaScript` is against

File: Makefile (hunk 1 above)
Comment (frite) [reply Makefile:11]: Good catch, thanks.

File: README.md (hunk 2 above)
Comment (frite) [reply README.md:14]: Totally, good catch as well.

File: Makefile (hunk 1 above)
Comment (frite) [reply Makefile]: Resolving as this is outdated.

File: README.md (hunk 2 above)
Comment (frite) [reply README.md]: Updated.

**Pull request merged**